pandas/tests/io/test_excel.py

```diff
@@ -995,11 +995,18 @@ def test_read_excel_nrows_greater_than_nrows_in_file(self, ext):
 
     def test_read_excel_nrows_non_integer_parameter(self, ext):
```

Comment:
Would be better to parametrize this test for cases like `[-1, 0, '1']` instead of having separate tests

Reply:
@WillAyd sure, I will make these changes as soon as we decide if we are keeping this PR :)